.github/workflows/bot-intermediate-assignment.yml

```
@@ -22,7 +22,7 @@ jobs:
     concurrency:
       group: intermediate-guard-${{ github.event.issue.number || github.run_id }}
       cancel-in-progress: false
-    if: contains(github.event.issue.labels.*.name, 'intermediate')
+    if: contains(github.event.issue.labels.*.name, 'skill: intermediate')
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)